package.json

```diff
@@ -75,7 +75,7 @@
         "@duckduckgo/autofill": "github:duckduckgo/duckduckgo-autofill#4.7.0",
         "@duckduckgo/content-scope-scripts": "github:duckduckgo/content-scope-scripts#2.1.1",
         "@duckduckgo/jsbloom": "^1.0.2",
-        "@duckduckgo/privacy-grade": "1.1.0",
+        "@duckduckgo/privacy-grade": "github:duckduckgo/privacy-grade#jkt/support-unknown-trackers",
```

Comment:
is this the correct branch still?

Reply:
No, we'd need to update it once the other PR merges :)